### ISSUE TITLE
Keyboard shortcut to go to angry creature

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1239]
When you get a message a creature is angry, there's very little you can do about it because it takes a very long time to find and identify the unhappy creature.

What would be very helpful to streamline this process is to be able to press a single key to instantly focus the screen on the last creature you got an alert about and when you press it again toggle through all the other unhappy creatures.

Bonus points if these creatures also show mood icons that indicate WHY they are unhappy so you can actually fix the problem.

**Comment by hwoarangmy** [comment]
Good idea. I guess we may need some more mood icons (to show that they are not happy because they don't have their fee especially)

**Comment by Loobinex** [comment]
Or you can use a single 'Angry' mood icon and switch back and forth between that and for example the 'get gold' icon.

**Comment by max-carroll** [comment]
Or How about there's some mechanism to cycle through the angry creatures, then that could be fired from a keypress or a button on the gui
